src/vstest.console/Processors/CLIRunSettingsArgumentProcessor.cs

```diff
@@ -160,6 +167,17 @@ private void CreateOrOverwriteRunSettings(IRunSettingsProvider runSettingsProvid
             }
         }
 
+        private bool TryUpdateTestRunParameterNode(IRunSettingsProvider runSettingsProvider, string node)
+        {
+            var match = runSettingsProvider.GetTestRunParameterNodeMatch(node);
```

Comment:
What happens when user gives invalid args like "TestRunParameters.Parameters(foo=bar)"

Reply:
validated this scenario